packages/client/src/schema/operators.ts

```diff
@@ -13,21 +13,51 @@ import { SelectableColumn } from './selection';
  */
 export const gt = <T extends ComparableType>(value: T): ComparableTypeFilter<T> => ({ $gt: value });
 
+/**
+ * Operator to restrict results to only values that are greater than the given value.
+ */
+export const greaterThan = <T extends ComparableType>(value: T): ComparableTypeFilter<T> => ({ $gt: value });
```

Comment:
```suggestion
export const greaterThan = gt;
```

What about doing this with aliases?

Reply:
Sure!